src/main/java/mate/dao/CarDaoImpl.java

```diff
@@ -204,7 +204,7 @@ private void deleteAllDriversExceptList(Car car) {
     }
 
     private List<Driver> getAllDriversByCarId(Long carId) {
-        String selectQuery = "SELECT id, name, license_number FROM cars_drivers cd "
+        String selectQuery = "SELECT id, name, license_number, login FROM cars_drivers cd "
```

Comment:
<img width="711" alt="Screenshot 2021-12-14 at 11 47 34" src="https://user-images.githubusercontent.com/33026471/146005833-62fcaa8c-1f2a-4ac0-8c8e-8817c66f5cbe.png">
let's also fetch password here, don't forget to set it to driver below